.github/workflows/test-r-scripts.yaml

```diff
@@ -63,13 +76,27 @@ jobs:
           r-version: ${{ matrix.config.r }}
           http-user-agent: ${{ matrix.config.http-user-agent }}
 
+      - name: Install system dependency of usethis >= 2.0.0
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt install libgit2-dev
```

Comment:
From usethis 2.0.0 there is a new system dependency.  usethis may not a dependency directly, but it is indirectly via devtools. So we need to install the system dependency or we'll fail to run the web tool scripts. 

Relates to https://github.com/2DegreesInvesting/docker/issues/28